docs/wiki/01_STUDY_PARAMETERS_SETUP/01_Study_Variables/PscID.md

```diff
@@ -0,0 +1,75 @@
+# PSCID (Project Study Center ID)
+
+The PSCID is the main participant identifier in Loris. With one ID assigned to each participant per project. It serves the purposes of this this this and this.
+
+The configuration of PSCID's is designed to allow for flexible naming and generation. Both naming and generation are set in a configuration file called `config.xml`. The following table will help to familiarize you with how these options generate different-looking PSCID's.
+
+|generation|examples|options|
+|:--|:--|:--|
+|sequential|MTL0006, MTL0007, MTL0008|Generates a static prefix with incrementing numbers|
+|manual|A1, B2, Q5|Generates a fixed-length alphanumeric pattern|
+|random|BUDDY29, BUDDY02, BUDDY96|Generates a static prefix followed by a random number of fixed length| 
```

Comment:
you are mixing generation type with format. i.e. sequential doesnt mean that ther eis a static prefix, thats an option the ADMIN makes, if they choose to have a static section then the config should clearly indicate it and this table doesnt because you are not showing the configuration associated with the example. I think you are better off removing this table and just relying on the content below it that explains it more clearly.

as for the "manual" part, well its manual, nothing gets generated.